CMakeLists.txt

```diff
@@ -373,8 +376,8 @@ if(NRN_ENABLE_MUSIC)
     message(FATAL "MUSIC requires -DNRN_ENABLE_PYTHON=ON")
   endif()
   find_package(Cython REQUIRED)
-  if(NOT Cython_FOUND)
```

Comment:
just curious. why do you remove this check? would you not want to keep it and add the one below in addition to it?